types/src/form.rs

```diff
@@ -173,6 +178,8 @@ pub fn is_root_morpheme(s: &str) -> bool {
 pub struct AnnotatedFormUpdate {
     /// Unique identifier of the form
     pub id: Uuid,
-    /// Original source text that can be either undefined or null
+    /// Original source text that could also be undefined
```

Comment:
the "could also be undefined" is not really needed. I would just add to the docstring for the struct that all fields are optional.

Reply:
Gotcha, made that change!